test/functional/forall/reduce-basic/CMakeLists.txt

```diff
@@ -175,25 +175,25 @@ unset( REDUCETYPES )
 #
 # If building a subset of openmp target tests, add tests to build here.
 #
-if(RAJA_ENABLE_TARGET_OPENMP)
-  if(RAJA_TEST_OPENMP_TARGET_SUBSET)
+# if(RAJA_ENABLE_TARGET_OPENMP)
```

Comment:
Please don't comment out test cases. Use a local variable in the CMakeLists.txt file to disable. There are other examples of this in the test directories.